.github/workflows/prev-version-integration.yaml

```diff
@@ -55,13 +55,6 @@ jobs:
           tunnelIdentifier: ${{ steps.create-job-id.outputs.uuid }}
       - name: Install Kite
         run: integration/js/script/install-kite
-      - name: Setup
-        # Temporarily we need to copy the run-test script over from 2.9.0. Can be removed after 2.10.0 is released
-        run: |
-          cp integration/js/script/run-test integration/js/script/run-test.bku
-          git checkout tags/v${{ steps.prev.outputs.prev_version }}
```

Comment:
We probably need this line to check out the previous version :D

Reply:
Yes, got it, thanks! I have kept the setup job with the checkout step now.